### PR TITLE
PROJQUAY-1614 - 3.4.2 release notes

### PR DESCRIPTION
**Pull request by thomasmckay**
**Issue:** https://issues.redhat.com/browse/PROJQUAY-1614

3.4.2 release notes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+
+### v3.4.2
+
+**Fixed**
+- Fix clair crash downloading RHEL content mapping
+- Quay config validates SMTP
```
Comment (kleesc) [inline CHANGELOG.md:6]: `config-tool`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+- Quay Operator mirror configurations cannot be decoded after QuayRegistry
+reconciliation
+- Fix encrypted passwords
```
Comment (kleesc) [inline CHANGELOG.md:9]: "config-tool now prevents `SECRET_KEY` from changing on config updates"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
+
+### v3.4.2
+
+**Fixed**
+- Fix clair crash downloading RHEL content mapping
+- Quay config validates SMTP
+- Quay Operator mirror configurations cannot be decoded after QuayRegistry
```
Comment (kleesc) [inline CHANGELOG.md:7]: Can you rephrase this. I'm not sure what changed from reading this line.

**Review by kleesc — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jonathankingfc) [reply CHANGELOG.md:9]: There wasn't a bug with encrypted passwords. The customer was not URL encoding their credentials with special characters. Closed the issue as "not a bug" so I don't think this needs to be in the changelog.

File: CHANGELOG.md (hunk 3 above)
Comment (thomasmckay) [reply CHANGELOG.md:7]: @alecmerdler do you have better wording?

Comment (alecmerdler) [reply CHANGELOG.md:7]: "Mirror configurations cannot be decoded when using Quay Operator"

**Review by alecmerdler — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (kleesc) [reply CHANGELOG.md]: @thomasmckay @jonathankingfc was there any actual change with SECRET_KEY in the config tool? If not we should just remove this.

**Review by kleesc — APPROVED** [review]
